src/caldavclient.cpp

```diff
@@ -261,7 +261,11 @@ QList<Settings::CalendarInfo> CalDavClient::loadCalendars(Accounts::Account *acc
         if (!enabledCalendars.contains(calendarPaths[i])) {
             continue;
         }
-        Settings::CalendarInfo info = { calendarPaths[i], displayNames[i], colors[i] };
+        // the calendar path may be percent-encoded.  Return UTF-8 QString.
+        QString remoteCalendarPath = calendarPaths[i].contains('%')
```

Comment:
Th check seems unnecessary, QUrl::fromPercentEncoding() should already check and not modify the string unnecessarily.